axelrod/strategies/titfortat.py

```diff
@@ -4,8 +4,16 @@
 
 
 class TitForTat(Player):
-    """A player starts by cooperating and then mimics previous move by opponent."""
+    """
+    A player starts by cooperating and then mimics previous move by
```

Comment:
Yes that's indeed what it should be. Away from a computer right now but
will make that change (and also change in the docs).

On Sun, 25 Oct 2015, 17:00 Marc Harper, PhD <notifications@github.com>
wrote:

> In axelrod/strategies/titfortat.py
> <https://github.com/Axelrod-Python/Axelrod/pull/383#discussion_r42947135>:
>
> > @@ -4,8 +4,16 @@
> >
> >
> >  class TitForTat(Player):
> > -    """A player starts by cooperating and then mimics previous move by opponent."""
> > +    """
> > +    A player starts by cooperating and then mimics previous move by
>
> Can we say "mimics the previous action of the opponent." ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Axelrod-Python/Axelrod/pull/383/files#r42947135>.
>